kernel/mythos/invocation/mythos/protocol/ExecutionContext.hh

```diff
@@ -125,6 +126,13 @@ namespace mythos {
         uintptr_t gs;
       };
 
+      struct SetPriority : public InvocationBase {
+        constexpr static uint16_t label = (proto<<8) + SET_PRIORITY;
+        SetPriority(bool priority)
+          : InvocationBase(label,getLength(this)), priority(priority) {}
+        bool priority;
+      };
```

Comment:
I'm not sure if we should limit that to a bool. See comment on interface. `setPriority(false)` could be hard to interpret.